src/platforms/common/data-management/debug-files/symbol-servers.mdx

```diff
@@ -280,6 +281,19 @@ Examples:
 - `wk/wkernel32.pdb/FF9F9F7841DB88F0CDEDA9E1E9BFF3B5A/wkernel32.pdb`
 - `KE/KERNEL32.dll/590285E9e0000/KERNEL32.dll`
 
+### Debuginfod
+
+Path : `<build id>/<type>`
```

Comment:
Even though this example shows the lack of `/buildid/` prefix, it might be good to call that out explicitly.

The expected debuginfod paths are like: `/buildid/BUILDID/debuginfo` and `/buildid/BUILDID/executable`.
See: https://man.archlinux.org/man/core/debuginfod/debuginfod.8.en#WEBAPI

This caught me by surprise yesterday, and the `/buildid/` prefix needs to be part of the symbol source `url` for this to function properly.

Reply:
Right. I was going by the path resolution function `get_debuginfod_path` in `symbolicator`, which assumes that the `/buildid` prefix is present outside of the individual path, as you say.

Do you think it's better to add a note about the prefix or to just incorporate it into the path directly?

Reply:
I would assume incorporating it into the path would be a breaking change. Especially now that I added a debuginfod source to sentry, there wouldn’t be a forward compatible way to migrate that :-(

Reply:
Sorry, I meant in the documentation here. I could change the above to
```suggestion
Path : `buildid/<build id>/<type>`
```

Reply:
Hm, I would still keep that as the path that symbolicator adds "on top" of the base `url`?

Reply:
Ok, I'll add a note then.